Microsoft.Toolkit.Uwp.Input.GazeInteraction/pch.h

```diff
@@ -0,0 +1,54 @@
+﻿//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#pragma once
+
+#include <collection.h>
+#include <ppltasks.h>
+
+//#define _USE_MATH_DEFINES
+//#include <math.h>
+#include <assert.h>
+
+//#include <Windows.Input.Gaze.h>
+
+using namespace Windows::Foundation;
+using namespace Windows::Devices::Enumeration;
+using namespace Windows::Devices::HumanInterfaceDevice;
+using namespace Windows::UI::Core;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Shapes;
+
+#include <strsafe.h>
+class Debug
+{
+public:
+    static void WriteLine(wchar_t *format, ...)
+    {
+        wchar_t message[1024];
+        va_list args;
+        va_start(args, format);
+        StringCchVPrintf(message, 1024, format, args);
+        OutputDebugString(message);
+        OutputDebugString(L"\n");
+    }
+};
+
+inline static TimeSpan operator + (const TimeSpan& lhs, const TimeSpan& rhs) { return TimeSpan{ lhs.Duration + rhs.Duration }; }
+inline static TimeSpan operator - (const TimeSpan& lhs, const TimeSpan& rhs) { return TimeSpan{ lhs.Duration - rhs.Duration }; }
+inline static TimeSpan operator * (int lhs, const TimeSpan& rhs) { return TimeSpan{ lhs*rhs.Duration }; }
+inline static bool operator < (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration < rhs.Duration; }
+inline static bool operator <= (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration <= rhs.Duration; }
+inline static bool operator > (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration > rhs.Duration; }
+inline static bool operator >= (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration >= rhs.Duration; }
+inline static bool operator == (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration == rhs.Duration; }
+inline static bool operator != (const TimeSpan& lhs, const TimeSpan& rhs) { return lhs.Duration != rhs.Duration; }
+
+static TimeSpan TimeSpanZero{ 0 };
+inline static TimeSpan TimeSpanFromMicroseconds(uint64 milliseconds) { return TimeSpan{ (int64)(10 * milliseconds) }; }
+inline static TimeSpan TimeSpanFromMicroseconds(int64 milliseconds) { return TimeSpan{ 10 * milliseconds }; }
+inline static TimeSpan TimeSpanFromMicroseconds(int milliseconds) { return TimeSpan{ 10 * milliseconds }; }
+
+#define BEGIN_NAMESPACE_GAZE_INPUT namespace Microsoft { namespace Toolkit { namespace Uwp { namespace Input { namespace GazeInteraction {
```

Comment:
Should we change namespace from `...Input.GazeInteraction` to just `...Input.Gaze` since that's the input type?

Reply:
The initial name was just `...Input.Gaze`. After much internal discussion, it was decided to rename it `Input.GazeInteraction`. I think that will likely remain. Can you please explain what you mean by "since that is the input type"

Reply:
@harishsk I meant The Input type is "Gaze" not "GazeInteracton". Similarly there's an `Input.Manipulations` not `Input.ManipulationInteraction`.

Reply:
@dotMorten Thanks for the clarification. Eye gaze can be used for interaction, analysis, attention etc. This library is used specifically for interaction. Hence the name GazeInteraction. Also, we have received very strong internal recommendation to not use the word by itself and hence we would like to stick with `GazeInteraction` as the name.